Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -35,6 +35,8 @@
 #include <cstdlib>
 #include <random>
 
+const int TENSILE_STREAMK_GRID 1
```

Comment:
don't we need ; here?

Reply:
Yea, I had this as a define while testing, started to change it before posting but must have messed up the saved version. I've changed it back to define.